content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -41,3 +41,21 @@ The Sigfox and Actility agents now support the configuration of multiple connect
 ![All devices](/images/release-notes/multiple_lns_connection.png)
 
 For details, refer to [Sigfox](https://cumulocity.com/guides/protocol-integration/sigfox) or [Actility LoRa](https://cumulocity.com/guides/protocol-integration/lora-actility) in the *Protocol integration guide*.
+
+### Device Management
+
+#### Revised LWM2M device registration
+
+The LWM2M agent now provides revised single- and device bulk registration wizards. These dialogs allows the user to conveniently perform single and bulk registrations of LWM2M devices. They also perform input validation and detect common errors before the device is created. See also [LWM2M > Registering LWM2M devices](https://cumulocity.com/guides/protocol-integration/lwm2m/##register-device) in the *Protocol integration guide*.
```

Comment:
single- and device bulk registration wizards -> single and bulk device registration wizards

Reply:
fixed